File: visual_basic/integracion_con_base_de_datos/cuardar_multiples_contactos/conexion.vb
Imports MySql.Data.MySqlClient
Module conexion
    Public Function ObtenerConexion() As MySqlConnection
        Dim cadenaConexion As String = "server=localhost;user id=root;password=;database=guardar_mis_multiples_contactos"
        Dim conexion As New MySqlConnection(cadenaConexion)
        Return conexion
    End Function
End Module

File: python/agenda_con_mas_campos/conexion.py
import mysql.connector
def conectar():
    try:
        conexion = mysql.connector.connect(
            host="localhost",
            user="root",
            password="",
            database="agenda_mas_campos"
        )
        return conexion
    except mysql.connector.Error as e:
        print("Error al conectar:", e)
        return None